irods_testing_environment/irods_config.py

```diff
@@ -105,6 +105,16 @@ def get_irods_version_info(container, version_file_key):
     raise RuntimeError(f'[{container.name}]: No iRODS version file found')
 
 
+def server_version_is_irods_5(container):
+    """Returns True if iRODS 5 is installed in the container.
+
+    Arguments:
+    container -- the container to check
+    """
+    irods5_binary_exists = "[ -f /usr/sbin/irodsAgent ]"
+    return execute.execute_command(container, irods5_binary_exists, user='irods') is 0
```

Comment:
Ah, didn't notice this before. Prefer `== 0` to `is 0`. `is` is used incorrectly... pretty much everywhere in this project. :( No time like the present.

Reply:
Oh right. I remember the warnings/suggestions from python about that.

Will fix.

Reply:
Done